Source/JavaScriptCore/wasm/WasmBBQJIT.cpp

```diff
@@ -4029,15 +4037,16 @@ void BBQJIT::slowPathRestoreBindings(const RegisterBindings& bindings)
     }
 }
 
-template<size_t N>
-void BBQJIT::saveValuesAcrossCallAndPassArguments(const Vector<Value, N>& arguments, const CallInformation& callInfo, const TypeDefinition& signature)
+template<typename Args>
+void BBQJIT::saveValuesAcrossCallAndPassArguments(const Args& arguments, const CallInformation& callInfo, const TypeDefinition& signature)
```

Comment:
Nit: You can just use `const auto& arguments`

Reply:
Changed.